core/storage/exploration/gae_models.py

```diff
@@ -201,6 +201,19 @@ class ExplorationRightsModel(base_models.VersionedModel):
     )
 
     def save(self, committer_id, commit_message, commit_cmds):
+        """Calls super class method 'commit' to save the version of the
+        exploration and update the model.
+
+        Args:
+            committer_id: str. The user_id of the user who committed the
+                change.
+            commit_message: str. The commit description message.
+            commit_cmds: list(dict). A list of commands, describing changes
+                made in this model, should give sufficient information to
+                reconstruct the commit. Each dict always contains:
+                    cmd: str. Unique command.
```

Comment:
What do you mean by "unique command"? It's not clear.

Reply:
`cmd` is the unique key here in the dict. I've updated the docstrings.